pyproject.toml

```diff
@@ -20,38 +20,40 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 license = { "text" = "Apache Software License" }
+
+# We can use newer setuptools after removing chainer support.
```

Comment:
It's better to make a comment for some packages that strictly require a specific version, e.g., `"sentencepiece==0.2.0",` (currently == or <=).

Also, what is the process of upgrading them?
When we fix the version, we can test upgrading to it. If it is successful, we will replace the versions with the latest ones.

Reply:
I think we can use Dependabot or GitHub Actions to automatically comment or create PR.

- Dependabot: Automatically checks `pyproject.toml` and creates PRs for outdated packages.

- GitHub Actions + Scripts: Post the list of outdated packages as a comment on PRs.